File: translations/codenet/mistral_commenetd/Go2J/gpt-4o/Code/s984400335.java
// Package declaration
import java.util.Scanner;

// Class declaration
public class Main {

    // Method to find the minimum number among given integers
    public static int min(int... nums) {
        // Initializing the minimum value with the first integer
        int min = nums[0];

        // Iterating through the array 'nums' to find the minimum value
        for (int v : nums) {
            // If the current value is less than the minimum value, update the minimum value
            if (v < min) {
                min = v;
            }
        }

        // Returning the minimum value
        return min;
    }

    // Main method declaration
    public static void main(String[] args) {
        // Creating a Scanner object for reading input values
        Scanner scanner = new Scanner(System.in);

        // Declaring three integer variables for reading input values
        int n, a, b;

        // Reading input values from the standard input
        n = scanner.nextInt();
        a = scanner.nextInt();
        b = scanner.nextInt();

        // Printing the minimum value between 'n*a' and 'b'
        System.out.println(min(n * a, b));

        // Closing the scanner
        scanner.close();
    }
}

// <END-OF-CODE>

File: translations/codenet/mistral_commenetd/Go2C/gpt-4o/Code/s984400335.c
// Including standard input/output library
#include <stdio.h>

// Function declaration for finding the minimum number among given integers
int min(int nums[], int size) {
    // Initializing the minimum value with the first integer
    int min = nums[0];

    // Iterating through the array 'nums' to find the minimum value
    for (int i = 1; i < size; i++) {
        // If the current value is less than the minimum value, update the minimum value
        if (nums[i] < min) {
            min = nums[i];
        }
    }

    // Returning the minimum value
    return min;
}

// Main function declaration
int main() {
    // Declaring three integer variables for reading input values
    int n, a, b;

    // Reading input values from the standard input using 'scanf' function
    scanf("%d %d %d", &n, &a, &b);

    // Declaring an array to hold the values for comparison
    int values[2] = {n * a, b};

    // Printing the minimum value between 'n*a' and 'b' using 'printf' function
    printf("%d\n", min(values, 2));

    return 0;
}

// <END-OF-CODE>
